app/models/github.rb

```diff
@@ -33,21 +33,33 @@ def create_new_commit(destination_path, data, message, author_name, author_email
         end
       end
 
-      def delete_file_at_path(destination_path, message, author_name, author_email)
+      def delete_file_at_path(destination_path, message, sha, author_name, author_email)
         CommitResponse.new do
           delete(File.join('contents', URI.escape(destination_path)),
-                 :message   => message,
-                 :branch    => BRANCH,
+                 :Message   => message,
```

Comment:
one of the really common key combos for file navigation is right next to the one that capitalises